src/amm.rs

```diff
@@ -1,13 +1,17 @@
-use crate::{client::Web3, jsonrpc::error::Web3Error};
+// Performs interactions with AMMs (Automated Market Makers) on ethereum
+use crate::{client::Web3, jsonrpc::error::Web3Error, types::SendTxOption};
 use clarity::{
-    abi::Token,
+    abi::{encode_call, Token},
     constants::{TT160M1, TT24M1},
-    Address, Uint256,
+    Address, PrivateKey, Uint256,
 };
+use num::BigUint;
 
 lazy_static! {
     pub static ref UNISWAP_QUOTER_ADDRESS: Address =
```

Comment:
These should have some doc comments indicating that they are addresses for production Ethereum versions of these contracts.

Reply:
The /// with the blank line does not come up as a doc comment for that var, so if I try to import it and look at the docs it won't include the line about the address being for production Ethereum.